gnosis/cowswap
# Update xdai and wxdai token logo

# Summary

Fixes #1179 
Before:
![image](https://user-images.githubusercontent.com/622217/149204240-1c9f6279-d524-42e0-9269-4d60ff8a4be4.png)
After:
![image](https://user-images.githubusercontent.com/622217/149204282-281b531d-4b83-4651-8240-341e051ff770.png)


  # To Test

1. Open the page [`swap`](https://pr2122--gpswapui.review.gnosisdev.com/)
2. Connect a wallet while on xDai network
3. Pick xDai and wxDai
4. Check the token logos. Should look like the screenshot above.
